### DISCUSSION TITLE
What are the semantics of favs / starred toots?

### DISCUSSION THREAD

**Discussion by rgiersig** [mastodon/mastodon#18201] [Q&A] [answered]
I'm wondering, what a faved toot means. Do I get any notifications for that toot, e.g. if somebody answers?
Sorry if that is a FAQ, is there a design document that explains such things?
Thanks, Roland

**Comment by Gargron** [comment, marked as answer]
The author gets notified you favourited it. Your favourite is visibile on the post. You can also browse all posts you favourited (but that list is just for you).

**Reply by rgiersig** [reply]
Thank you. Is the notification persistent? Meaning, if I fav a toot and immediately remove the fav, will the author still be notified? Or must the fav be active when the authors client refreshes the notifications?

**Reply by ClearlyClaire** [reply]
In Mastodon, notifications get deleted when the object that caused it gets deleted. But that may not be the case on other software compatible with Mastodon.

**Reply by rgiersig** [reply]
I think I understand. 
So setting the fav causes a "is now faved" notification to be sent out, causing a notification message to be generated at the tooters end. 
And if the fav is removed again, a "no longer faved" notification is sent out. 
Makes sense. Thank you!

**Comment by trwnh** [comment]
semantically, it is a Like on an object. you are indicating to the author that you have seen the post and appreciate it in some way.

there is a user guide at docs.joinmastodon.org that may be useful

**Reply by rgiersig** [reply]
Thank you.
